modules/@apostrophecms/piece-type/ui/apos/components/AposDocsManager.vue

```diff
@@ -180,6 +181,9 @@ export default {
         emoji: '📄'
       };
     },
+    manuallyPublished() {
+      return this.options.localized && !this.options.autopublish;
+    },
```

Comment:
Should this be part of the publishMixin? the same computed just landed in AposDocsManager